src/pages/Home.tsx

```diff
@@ -34,24 +104,65 @@ const Home: () => JSX.Element = () => {
             height={['100%', null, '100%', null]}
             borderRadius={6}
           >
-            {currentSensor ? (
-              <AqiDial currentAqi={currentSensor} />
-            ) : (
-              <Text marginTop={[null, null, '20%', null]}>
-                {t('noActiveSensor')}
-              </Text>
+            {isMobile && (
+              <Box>
+                <IconButton
+                  position="relative"
+                  float="right"
+                  size="md"
+                  aria-label="Toggle Aqi Gauge"
```

Comment:
Since `aria-label`s are user facing, this should be externalized and translated. Also capitalized AQI because user facing. There is a similar one above and below that I didn't explicitly comment on but that this same change would be for too